doc/source/_static/GafferLogo.svg

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Rather than adding the logos directly, can we just copy them in-place using a generate script? They're already in the repo under `<localClone>/resources/`, so we should be able to copy them during the doc build itself.

Reply:
I wasn't sure if there was ever going to be a point where they might differ, so for now I just made them static. If you think it's unlikely, should it be added to SCons or its own generate.py script?

Reply:
I'd say its own generate.sh script (simple cp command). If you want to add that now, I can rebase my other branch top of this and adjust it to the new style generate for you.